Portal/src/Datahub.Core/Components/PageBanner/DHPageAlertDialog.razor

```diff
@@ -1,40 +1,52 @@
-﻿<MudDialog>
+﻿<MudDialog Style="height: 830px" ContentStyle="height: 100%">
```

Comment:
Should be okay here as we have no screen requirements, but it might make sense to have it set a min and max height based on the `vh` measurement